WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -57,6 +56,20 @@ class HelpAndSupportViewController: UIViewController {
     ///
     var displaysDismissAction = false
 
+    /// Custom help center web page related properties
+    /// If non-nil this web page is launched instead of Zendesk
+    ///
+    private let customHelpCenterContent: CustomHelpCenterContent?
+
+    init?(customHelpCenterContent: CustomHelpCenterContent, coder: NSCoder) {
```

Comment:
nit: if `customHelpCenterContent` has an optional type, is the initializer `required init?(coder: NSCoder)` below still needed since it just calls its super?

Reply:
I am afraid that we need that as it is a "required initializer". We have to implement that in the subclass as we have a custom `init` method declared. 
Kindly let me know if you know of a way to remove it.